src/services/balancer/contracts/contracts/liquidity-gauge.ts

```diff
@@ -2,23 +2,60 @@ import LiquidityGaugeAbi from '@/lib/abi/LiquidityGaugeV5.json';
 import { Multicaller } from '@/lib/utils/balancer/contract';
 import { configService } from '@/services/config/config.service';
 import { rpcProviderService } from '@/services/rpc-provider/rpc-provider.service';
+import { BigNumber } from '@ethersproject/bignumber';
+import { AddressZero, MaxUint256 } from '@ethersproject/constants';
 import { web3Service } from '@/services/web3/web3.service';
 import { Contract } from '@ethersproject/contracts';
+import { default as ERC20ABI } from '@/lib/abi/ERC20.json';
+
+const MAX_REWARD_TOKENS = 8;
 
 export class LiquidityGauge {
   instance: Contract;
 
   constructor(
-    public readonly address: string,
-    private readonly abi = LiquidityGaugeAbi,
+    public readonly address: string = '0x5be3bbb5d7497138b9e623506d8b6c6cd72daceb',
     private readonly provider = rpcProviderService.jsonProvider,
+    private readonly abi = LiquidityGaugeAbi,
```

Comment:
Please revert this to how it is on `develop`. The default address was set like that initially when I set this up for testing, there def shouldn't be a default here.

Reply:
Ok, but provider should still be the second param. Why would we need to change the abi? That's pretty static. The provider however, is subject to change sometimes.